Apigee/ManagementAPI/Developer.php

```diff
@@ -408,18 +451,21 @@ protected static function loadFromResponse(Developer &$developer, array $respons
         $developer->firstName = $response['firstName'];
         $developer->lastName = $response['lastName'];
         $developer->userName = $response['userName'];
-        $developer->organizationName = $response['organizationName'];
         $developer->status = $response['status'];
         $developer->attributes = array();
         if (array_key_exists('attributes', $response) && is_array($response['attributes'])) {
-            foreach ($response['attributes'] as $attribute) {
-                $developer->attributes[$attribute['name']] = @$attribute['value'];
+            foreach ($response['attributes'] as $key => $attribute) {
+                if (is_array($attribute)) {
+                    $developer->attributes[$attribute['name']] = @$attribute['value'];
+                } else {
+                    $developer->attributes[$key]= $attribute;
+                }
             }
         }
         $developer->createdAt = $response['createdAt'];
         $developer->createdBy = $response['createdBy'];
-        $developer->modifiedAt = $response['lastModifiedAt'];
-        $developer->modifiedBy = $response['lastModifiedBy'];
+        $developer->modifiedAt = array_key_exists('lastModifiedAt', $response) ? $response['lastModifiedAt'] : $response['modifiedAt'];
```

Comment:
In which circumstances we get back "modifiedAt" instead of "lastModifiedAt"?

Reply:
It's different on the object and on the response... I had to make sure it is compatible.